src/TestFileSupport.cpp

```diff
@@ -0,0 +1,59 @@
+#include "TestFileSupport.hpp"
+#include <system_error>
+#include <cerrno>
+
+bool TestFileSupport::CreateTestFiles(std::string inputfiles_directory)
```

Comment:
Too long function. Some elements should be extracted into smaller functions.

Reply:
I made shorter names: 
- InputFile::Create and InputFile::Remove.
Done.